src/pages/company-info/index.js

```diff
@@ -0,0 +1,206 @@
+import { Box, Grid } from '@mui/material'
+import { useFormik } from 'formik'
+import React, { useContext, useEffect, useState } from 'react'
+import CustomImage from 'src/components/Inputs/CustomImage'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import FormShell from 'src/components/Shared/FormShell'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import useResourceParams from 'src/hooks/useResourceParams'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import toast from 'react-hot-toast'
+import { useForm } from 'src/hooks/form'
+
+const CompanyInfo = () => {
+  const [initialValues, setInitialData] = useState({
+    plantId: '',
+    accountId: JSON.parse(window.sessionStorage.getItem('userData')).accountId,
+    name: '',
+    webSite: '',
+    taxNo: '',
+    licenseNo: '',
+    crNo: '',
+    logoUrl: '',
+    flName: '',
+    attachment: null
+  })
+  const { getRequest, postRequest } = useContext(RequestsContext)
+
+  const { labels: labels, access: maxAccess } = useResourceParams({
+    datasetId: ResourceIds.CompanyInfo
+  })
+
+  useEffect(() => {
+    getData()
+  }, [])
+
+  async function getData() {
+    const res = await getRequest({
+      extension: SystemRepository.CompanyInfo.get,
+      parameters: `_filter=`
+    })
+    res.record.accountId = JSON.parse(window.sessionStorage.getItem('userData')).accountId
```

Comment:
No references to session storage. Please check a file named storage.js for using sessionstorage.